qa/L0_http/generate_endpoint_test.py

```diff
@@ -71,6 +71,9 @@ def generate_expect_failure(self, model_name, inputs, msg):
         r = requests.post(
             url, data=inputs if isinstance(inputs, str) else json.dumps(inputs)
         )
+        # Content-Type header should always be JSON for errors
+        self.assertEqual(r.headers["Content-Type"], "'application/json'")
```

Comment:
Can we add a note to the generate endpoint documentation with that clarification (for initial errors content type will be application/json). Another option could be to use text/event-stream for all errors as well. I could not find a reference that spelled it out one way or the other.